linux/omero-web-systemd.service

```diff
@@ -5,11 +5,12 @@ Requires=nginx.service
 [Service]
 User=omero
 Type=forking
-PIDFile=/home/omero/OMERO.server/var/django.pid
+#PIDFile=/home/omero/omero/var/django.pid
 Restart=on-failure
 RestartSec=10
-ExecStart=/home/omero/OMERO.server/bin/omero web start
-ExecStop=/home/omero/OMERO.server/bin/omero web stop
+Environment="OMERODIR=/home/omero/OMERO.server"
+ExecStart=/home/omero/omeroenv/bin/omero web start
```

Comment:
The ansible roles install to `/opt/omero/{web,server}/venv3`. Should we start to unify?

Reply:
We should probably unify to make the transition easier.
Note that I have kept "web installation" in this repo mainly for internal use.
This does not appear in the doc since we use omeroweb-install in the doc